src/main/java/ru/itmo/roguelike/characters/Player.java

```diff
@@ -290,15 +292,10 @@ public void addExp(float exp) {
 
         if (levelGain > 0) {
             if (level <= 2 && level + levelGain > 2) {
-                IntCoordinate bossPosition = new IntCoordinate(position);
-                bossPosition.add(new IntCoordinate(100, 100));
-
-                Enemy.builder(PersonX::new)
-                        .setPosition(bossPosition)
-                        .setBehavior(MobWithTarget.builder(AggressiveBehavior::new))
-                        .setRadius(100000)
-                        .setTarget(this)
-                        .createAndRegister();
+                eventManager.add(() -> {
+                    bossManager.createBoss();
+                    return false;
+                });
```

Comment:
```suggestion
            bossManager.createBoss();
```
Это, вроде, то же самое

Reply:
🤦